cidr-filtering-cni

```diff
@@ -235,8 +235,12 @@ main() {
         mkdir -p /var/run/netns/
         ln -sfT "$CNI_NETNS" /var/run/netns/"${CNI_CONTAINERID}"
 
+        local kubeconfig_path="/etc/cni/net.d/cidr-filtering.d/cidr-filtering.kubeconfig"
+        if [[ -f "/etc/kubernetes/cni/net.d/cidr-filtering.d/cidr-filtering.kubeconfig" ]]; then
+            kubeconfig_path="/etc/kubernetes/cni/net.d/cidr-filtering.d/cidr-filtering.kubeconfig"
+        fi
```

Comment:
Wouldn't it be preferable to have an override-able default ? I.e. your openshift daemonset spec would override the default.

This would enable this CNI to be re-used in other ecosystems.

Reply:
Done, not sure if this is the most elegant solution.
Maybe a better alternative could be having a directory in a constant place